cmd/soroban-rpc/internal/jsonrpc.go

```diff
@@ -25,6 +25,11 @@ import (
 	"github.com/stellar/soroban-tools/cmd/soroban-rpc/internal/transactions"
 )
 
+// maxHTTPRequestSize defines the largest request that the http
+// would be willing to accept. The implementation uses the default
+// MaxBytesHandler to limit the request size.
```

Comment:
I think that the comment is a bit redundant. The identifier is self-descriptive.

Regardless, what do you mean by _the http_ ? You mean _the http *server*_?

Reply:
I think that the const name is quite self-explanatory.. the size of the request that's being passed over the http protocol. Clearly, that includes some http-specific artifacts, like http headers, and such.

Reply:
oh - I see. you meant the "http" in the variable description. yes, I'll update that.